rspace/src/main/scala/coop/rchain/rspace/Store.scala

```diff
@@ -0,0 +1,303 @@
+package coop.rchain.rspace
+
+import java.nio.ByteBuffer
+import java.security.MessageDigest
+
+import cats.implicits._
+import com.google.protobuf.ByteString
+import com.trueaccord.scalapb.GeneratedMessage
+import coop.rchain.rspace.Serialize.mkProtobufInstance
+import coop.rchain.rspace.datamodels._
+import coop.rchain.rspace.internal.{Datum, Row, WaitingContinuation}
+import coop.rchain.rspace.util.{dropIndex, ignore, optionalWithResource}
+
+trait KeyValueAmbry {
+  private[rspace] type H = ByteBuffer
+
+  private[rspace] type T
+
+  trait KeyValueTable {
+    def iterateKeys(txn: T): Iterator[H]
+    def put(txn: T, keyBuffer: H, valueBuffer: ByteBuffer): Unit
+    def get(txn: T, keyBuffer: H): Option[ByteBuffer]
+    def delete(txn: T, keyBuffer: H): Unit
+    def isEmpty(txn: T): Boolean
+    def drop(txn: T): Unit
+  }
+
+  val _dbKeys: KeyValueTable
+  val _dbAs: KeyValueTable
+  val _dbPsKs: KeyValueTable
+  val _dbJoins: KeyValueTable
+}
+
+/**
+	* The main store class.
+	*
+	* To create an instance, use [[LMDBStore.create]].
+	*/
```

Comment:
Is it just github or the indentation is off here?

Reply:
Don't know exactly - after loading the code and running scalafmt git says that file unchanged.